spec/models/admin_user_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe AdminUser, :type => :model do
+
+  it 'should have a valid factory' do
+    expect(build_stubbed :admin_user).to be_valid
+  end
+
+  it { is_expected.to validate_presence_of :email }
+  it { is_expected.to validate_presence_of :password }
+  it { is_expected.not_to validate_presence_of :password_confirmation }
+  it { is_expected.to validate_uniqueness_of :email }
+  it { is_expected.not_to validate_uniqueness_of :password }
+  it { is_expected.not_to validate_uniqueness_of :password_confirmation }
```

Comment:
In fact, if I'm reading Devise docs correctly, we can remove all tests except the one for the factory. `AdminUser` includes Devise's `validatable` module which validates email presence, uniqueness & format, as well as password presence, length (8-72) & confirmation. A single test to ensure that the module is included is sufficient; more than that and we are testing the gem.